tests/test_fit_url.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
+import sys
+
+sys.path.append("../tidy3d_client_revamp")
```

Comment:
we shouldn't have this ideally.  In tests/__init__.py we have the importing of the tidy3d package.
https://github.com/flexcompute/tidy3d/blob/develop/tests/__init__.py#L3
so can you see if removing this line still works?  also the name of the repo has changed.

Reply:
It is now removed, and it is still working.